### PR TITLE
fix: resolve token ens name in EIP-681 links from qr code

### LINKED ISSUE #10345: Token ENS name is not resolved in EIP-681 links
# Bug Report

## Problem
Leftover  after https://github.com/status-im/status-react/pull/10280 - ENS name for token is not resolved (i.e.
ethereum:snt.thetoken.eth/transfer?address=3esmit.stateofus.eth&uint256=1337e17)
#### Expected behavior
asset is SNT

#### Actual behavior
error is shown

### Reproduction

- Open Status
- Go to wallet
- Scan QR with link `ethereum:snt.thetoken.eth/transfer?address=3esmit.stateofus.eth&uint256=1337e17`

### Additional Information
- Status version: nightly 16/04/2020
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by yqrashawn**
fix #10345

check #10345 for reproduction

https://user-images.githubusercontent.com/15090582/188836436-0517f976-f018-48f2-91f6-f3fefecd5b9a.mp4

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/qr_scanner/views.cljs [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-            [status-im.ui.components.topbar :as topbar]
             [status-im.ui.components.react :as react]
+            [status-im.ui.components.topbar :as topbar]
             [status-im.ui.screens.qr-scanner.styles :as styles]
-            [quo.design-system.colors :as colors]
-            [status-im.utils.config :as config]
-            [quo.core :as quo]
-            [reagent.core :as reagent]
-            ["react-native-camera-kit" :refer (CameraKitCamera)]))
+            [status-im.utils.config :as config]))
```
Comment (yqrashawn) [inline src/status_im/ui/screens/qr_scanner/views.cljs:13]: reorder only

File: src/status_im/ui/screens/qr_scanner/views.cljs [hunk 2]
```diff
@@ -48,7 +48,7 @@
        {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-cancel opts])}
        "Cancel"]
       [quo/button
-       {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-success opts (when-let [text @text-value] (string/trim text))])}
+       {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-success opts (when-let [text @text-value] (-> text string/trim (string/replace #"^Ethereum:" "ethereum:")))])}
```
Comment (yqrashawn) [inline src/status_im/ui/screens/qr_scanner/views.cljs:51]: This is for `qr-test-view` only
it's annoying that iOS input capitalize the first char

File: src/status_im/ethereum/eip681_test.cljs [hunk 3]
```diff
@@ -141,12 +144,14 @@
...
-                                                                                                                      :symbol   :SNT
-                                                                                                                      :decimals 18}})]
+  (let [{:keys [value symbol address]}
+        (eip681/extract-request-details
+         {:address       "0x744d70fdbe2ba4cf95131626614a1763df805b9e" :chain-id           1
+          :function-name "transfer"                                   :function-arguments {:uint256 1000 :address "0x89205a3a3b2a69de6dbf7f01ed13b2108b2c43e7"}}
+         {"0x744d70fdbe2ba4cf95131626614a1763df805b9e" {:address  "0x744d70fdbe2ba4cf95131626614a1763df805b9e"
+                                                        :name     "Status Network Token"
+                                                        :symbol   :SNT
+                                                        :decimals 18}})]
```
Comment (yqrashawn) [inline src/status_im/ethereum/eip681_test.cljs:154]: format only

**Review by flexsurfer — APPROVED** [review]

**Comment by qoqobolo** [comment]
@yqrashawn thanks for your work!
PR is tested and can be merged.

**Pull request merged**